include/boost/geometry/algorithms/detail/is_simple/areal.hpp

```diff
@@ -38,54 +37,34 @@ namespace detail { namespace is_simple
 {
 
 
-template <typename Ring, typename CSTag>
-struct is_simple_ring
+template <typename Ring, typename Strategy>
+inline bool is_simple_ring(Ring const& ring, Strategy const& strategy)
 {
-    static inline bool apply(Ring const& ring)
-    {
-        simplicity_failure_policy policy;
-        return ! boost::empty(ring)
-            && ! detail::is_valid::has_duplicates
-                    <
-                        Ring, geometry::closure<Ring>::value, CSTag
-                    >::apply(ring, policy);
-    }
-};
-
-
-template <typename Polygon, typename CSTag>
-class is_simple_polygon
-{
-private:
-    template <typename InteriorRings>
-    static inline
-    bool are_simple_interior_rings(InteriorRings const& interior_rings)
-    {
-        return
-            detail::check_iterator_range
+    simplicity_failure_policy policy;
+    return ! boost::empty(ring)
+        && ! detail::is_valid::has_duplicates
                 <
```

Comment:
one more space here ?

Reply:
The rationale is as follows: the number of spaces is the first multiple of 4 starting after the start of the struct's name. AFAIU you're saying that it should start exactly 4 spaces after the start of the struct's name. It requires less number of keys pressed to do it my way (1 or 2 tabs after enter vs tabs followed by spaces or backspaces) and it looks almost the same. But I'm flexible. ;)

Reply:
I am ok with this rationale